compliance/compliance-api/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Deserialize, Serialize};
+
+/// The request type for a compliance check
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ComplianceCheckRequest {
+    /// The wallet id to request
+    pub wallet_id: String,
```

Comment:
Should this be an address instead of a wallet ID? Also, are we going to be using this request type, or are we going to be parsing the wallet address from the route (as you have it currently configured)?

Reply:
oops yes, thanks